include/cantera/thermo/CoverageDependentSurfPhase.h

```diff
@@ -0,0 +1,476 @@
+ *        + \int_{298}^{T}cp_k^{o,cov}(T,\theta)dT}_{\text{coverage dependence}}
+ *
+ * \f]
+ *
+ * \f[
+ *  s_k^o(T,\theta)
+ *     = \underbrace{s_k^{o,ideal}(T)
+ *       + \int_{298}^{T}\frac{cp_k^{o,ideal}(T)}{T}dT}_{\text{low-coverage limit}}
+ *       + \underbrace{s_k^{o,cov}(T,\theta)
+ *       + \int_{298}^{T}\frac{cp_k^{o,cov}(T,\theta)}{T}dT}_{\text{coverage
+ *         dependence}}
+ * \f]
+ *
+ * \f[
+ *  cp_k^o(T,\theta)
+ *      = cp_k^{o,ideal}(T) + cp_k^{o,cov}(T,\theta)
+ * \f]
+ *
+ * ## Mathematical Models for Coverage-dependent Correction Terms
+ *
+ * Coverage-dependent correction terms for enthalpy and entropy can be calculated
+ * with one of the four algebraic models: linear dependecy model, piecewise-linear
+ * dependency model, polynomial dependency model, and interpolative dependency model.
+ * In the dependency model equations, a coverage-dependent correction term is denoted
+ * by \f$ f^{cov} \f$ where \f$ f \f$ can be either enthalpy or entropy. Because
+ * lateral interaction can compose of both self- and cross- interactions, the total
+ * correction term of species \f$ k \f$ is a sum of all interacting species \f$ j \f$
+ * which can include itself. Coefficients \f$ c1_{k,j}-c7_{k,j} \f$ are user-provided
+ * parameters that can be given in input mechanism.
+ *
+ * Linear dependency model:
+ * \f[
+ *  f^{cov}_k(\theta) = \sum_j c1_{k,j} \theta_j
+ * \f]
+ *
+ * Piecewise-linear dependency model:
+ * \f[
+ * f^{cov}_k(\theta) = \sum_j \left\{
+ *  \begin{array}{ll}
+ *  c2_{k,j}\theta_j & \text{, } \theta_j \leq \theta^\text{change}_{k,j} \\
+ *  \left[c3_{k,j}(\theta_j - \theta^\text{change}_{k,j})
+ *  + (c2_{k,j}\theta^\text{change}_{k,j})\right]
+ *  & \text{, } \theta_j > \theta^\text{change}_{k,j} \\
+ *  \end{array}
+ *  \right.
+ * \f]
+ *
+ * Polynomial dependency model:
+ * \f[
+ *  f^{cov}_k(\theta) =
+ *   \sum_j \left[c4_{k,j}\theta_j + c5_{k,j}\theta_j^2
+ *           + c6_{k,j}\theta_j^3 + c7_{k,j}\theta_j^4\right]
+ * \f]
+ *
+ * Interpolative dependency model:
+ * \f[
+ *  f^{cov}_k(\theta) =
+ *   \sum_j \left[\frac{f^{cov}_k(\theta^{higher}_j) - f^{cov}_k(\theta^{lower}_j)}
+ *   {\theta^{higher}_j - \theta^{lower}_j}(\theta_j - \theta^{lower}_j)
+ *   + f^{cov}_k (\theta^{lower}_j)\right] \text{, }
+ *   \theta^{lower}_j \leq \theta_j < \theta^{higher}_j
+ * \f]
+ *
+ * Coverage-dependent heat capacity is calculated using an equation with a
+ * quadratic dependence on coverages and a logarithmic dependence on temperature.
+ * Temperature is nondimesionalized with a reference temperature of 1 K.
+ * The coverage-dependent heat capacity of species \f$ k \f$ is a sum of
+ * all quantities dependent on coverage of species \f$ j \f$. Coefficients
+ * \f$ c8_{k,j} \text{ and } c9_{k,j} \f$ are user-provided parameters that can be
+ * given in input mechanism.
+ *
+ * \f[
+ *  cp^{cov}_k(\theta) =
+ *   \sum_j \left(c8_{k,j} ln\left(\frac{T}{1\text{ K}}\right)
```

Comment:
Check for this elsewhere:
```suggestion
 *   \sum_j \left(c8_{k,j} \ln\left(\frac{T}{1\text{ K}}\right)
```